src/standalone/Vegetation/Canopy.jl

```diff
@@ -47,6 +47,278 @@ include("./canopy_parameterizations.jl")
 using Dates
 include("./autotrophic_respiration.jl")
 include("./spatially_varying_parameters.jl")
+
+########################################################
+# Convenience constructors for Canopy model components
+########################################################
+
+## Autotrophic respiration models
+"""
+    AutotrophicRespirationModel{FT}() where {FT <: AbstractFloat}
+
+Creates a AutotrophicRespirationModel using default parameters of type FT.
+"""
+function AutotrophicRespirationModel{FT}() where {FT <: AbstractFloat}
+    parameters = AutotrophicRespirationParameters(FT)
+    return AutotrophicRespirationModel{FT, typeof(parameters)}(parameters)
+end
+
+## Energy models
+"""
+    BigLeafEnergyModel{FT}(; ac_canopy = FT(2e3)) where {FT <: AbstractFloat}
+
+Creates a BigLeafEnergyModel using default parameters of type FT.
+
+The following default parameter is used:
+- ac_canopy = FT(2e3) (J m^-2 K^-1) - canopy specific heat per area
+"""
+function BigLeafEnergyModel{FT}(;
+    ac_canopy::FT = FT(2e3),
+) where {FT <: AbstractFloat}
+    parameters = BigLeafEnergyParameters{FT}(ac_canopy)
+    return BigLeafEnergyModel{FT, typeof(parameters)}(parameters)
+end
+
+## Photosynthesis models
+"""
+    FarquharModel{FT}(
+        is_c3::MECH;
+        Vcmax25::VC = 5e-5,
+        pc::FT = -2e6,
+        sc::FT = 5e-6,
+    ) where {
+        FT <: AbstractFloat,
+        MECH <: Union{FT, ClimaCore.Fields.Field},
+        VC <: Union{FT, ClimaCore.Fields.Field},
+    }
+
+Creates a FarquharModel using default parameters of type FT.
+
+The following default parameters are used:
+- Vcmax25 = 5e-5 (mol m^-2 s^-1) - maximum carboxylation rate at 25C
+- sc = 5e-6 (Pa^{-1}) - sensitivity to low water pressure in the moisture stress factor [Tuzet et al. (2003)]
+- pc = -2e6 (Pa) - reference water pressure for the moisture stress factor [Tuzet et al. (2003)]
+"""
+function FarquharModel{FT}(;
+    is_c3::MECH,
```

Comment:
I think we would also pass the domain here, and then get Vcmax25 and is_c3 from: https://github.com/CliMA/ClimaLand.jl/blob/04effc88b7cb9e1c3b2262e5be198b6782d03a95/src/standalone/Vegetation/spatially_varying_parameters.jl#L147